contrib/cirrus/lib.sh

```diff
@@ -244,6 +244,21 @@ use_netavark() {
     # N/B: The CNI packages are still installed and available. This is
     # on purpose, since CI needs to verify the selection mechanisms are
     # functional when both are available.
+
+    # See ./contrib/cirrus/CIModes.md.
+    # Vars defined by cirrus-ci
+    # shellcheck disable=SC2154
+    if [[ "$CIRRUS_PR_DRAFT" == "true" ]]; then
+        # shellcheck disable=SC2154
+        if [[ "$CIRRUS_CHANGE_TITLE" =~ CI:UPDATE_NVAV ]]; then
+            warn "Updating netavark/aardvark packages.]"
+            showrun dnf update -y netavark aardvark-dns
+        elif [[ "$CIRRUS_CHANGE_TITLE" =~ CI:LATEST_NVAV ]]; then
+            warn "Installing latest netavark/aardvark packages."
+            showrun dnf copr enable rhcontainerbot/podman-next -y
+            showrun dnf upgrade -y netavark aardvark-dns
+        fi
+    fi
```

Comment:
Didn't get a proper chance to review today, so am filing first-impressions only: this disturbs me. It will cause confusion down the line. I believe it should look more like:
```
if CIRRUS_CHANGE_TITLE looks like this-or-that
    if not CIRRUS_PR_DRAFT
        die "Magic AVNV string can only be used on DRAFT PRs"
    fi

    ....now it's OK to check
```
I also think the `UPDATE` and `LATEST` strings are going to cause even more confusion: I _think_ the remember-trick is "UPDATE means dnf-update, the other one means copr"?? There has to be a more intuitive way to do this but I'm giving up for the day. Maybe `CI:NVAV=COPR` and `CI:NVAV=DNF`? Anyhow, bye for now.